lib/resources.rb

```diff
@@ -143,8 +143,18 @@ def disclaimer
         "guaranteed to yield the results shown on the site.</p>\n"
       end
 
```

Comment:
It was mentioned we should have some documentation around this method and it’s use.

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/608/commits/2f931f0c55b754fe1337fe568a816bdbaaf4bef1)

Reply:
updated the return [here](https://github.com/cerner/fhir.cerner.com/pull/608/commits/ec79347b403e986ef8d8f5a04db9805cc64ad3e0#diff-3353fe6ded4833be03f1bb156f58f64c48c1295ef3657fe3d208e598784de1c0R151)